libcontainer/container_linux.go

```diff
@@ -100,7 +100,7 @@ type Container interface {
 	// Restore restores the checkpointed container to a running state using the criu(8) utility.
 	Restore(process *Process, criuOpts *CriuOpts) error
 
-	// If the Container state is RUNNING or CREATED, sets the Container state to PAUSING and pauses
+	// If the Container state is RUNNING or CREATED, sets the Container state to PAUSED and pauses
```

Comment:
Although the actual state that `Pause` internally sets is `FROZEN`, I hope this should be okay as an externally exposed func doc? cc/ @kolyshkin

Reply:
Yes, it's "frozen" in cgroup lingo, and "paused" in libcontainer lingo.